src/openklant/conf/api.py

```diff
@@ -14,35 +14,27 @@
 REST_FRAMEWORK[
     "DEFAULT_PAGINATION_CLASS"
 ] = "rest_framework.pagination.PageNumberPagination"
+REST_FRAMEWORK["DEFAULT_SCHEMA_CLASS"] = "drf_spectacular.openapi.AutoSchema"
+
 SECURITY_DEFINITION_NAME = "JWT-Claims"
 
-SWAGGER_SETTINGS = BASE_SWAGGER_SETTINGS.copy()
-
-SWAGGER_SETTINGS.update(
-    {
-        "DEFAULT_INFO": "openklant.components.legacy.klanten.api.schema.info",
-        "SECURITY_DEFINITIONS": {
-            SECURITY_DEFINITION_NAME: {
-                # OAS 3.0
-                "type": "http",
-                "scheme": "bearer",
-                "bearerFormat": "JWT",
-                # not official...
-                # 'scopes': {},  # TODO: set up registry that's filled in later...
-                # Swagger 2.0
-                # 'name': 'Authorization',
-                # 'in': 'header'
-                # 'type': 'apiKey',
-            }
-        },
-        "DEFAULT_FIELD_INSPECTORS": (
-            "vng_api_common.inspectors.fields.HyperlinkedIdentityFieldInspector",
-            "vng_api_common.inspectors.fields.HyperlinkedRelatedFieldInspector",
-            "openklant.components.utils.inspectors.ExpandSerializerInspector",
-        )
-        + BASE_SWAGGER_SETTINGS["DEFAULT_FIELD_INSPECTORS"],
-    }
-)
+SPECTACULAR_SETTINGS = {
+    "SERVE_INCLUDE_SCHEMA": False,
+    "CAMELIZE_NAMES": True,
```

Comment:
I saw we generate paths including the `v1` part, this will remove that:
```suggestion
    "CAMELIZE_NAMES": True,
    "SCHEMA_PATH_PREFIX": r"/v[0-9]+",
    "SCHEMA_PATH_PREFIX_TRIM": True,
```